samples/matter/light_switch/src/light_switch.h

```diff
@@ -25,8 +27,8 @@ class LightSwitch {
 		Off /* Turn off light on lighting-app device */
 	};
 
-	void Init(chip::EndpointId aLightSwitchEndpoint);
-	void InitiateActionSwitch(Action);
+	void Init(chip::EndpointId LightSwitchEndpoint);
+	void InitiateActionSwitch(Action mAction);
```

Comment:
```suggestion
	void InitiateActionSwitch(Action action);
```
to avoid confusing with members.